plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/ExpEmbedding.kt

```diff
@@ -30,36 +34,41 @@ sealed interface IntArithExpression : ExpEmbedding {
 data class Add(
     override val left: ExpEmbedding,
     override val right: ExpEmbedding,
+    override val pos: KtSourceElement? = null,
 ) : IntArithExpression {
-    override fun toViper() = Exp.Add(left.toViper(), right.toViper())
+    override fun toViper() = Exp.Add(left.toViper(), right.toViper(), Position.KtSourcePosition(pos))
```

Comment:
Here and elsewhere: you are potentially passing `null` as the position, should we not be passing `NoPosition` in those cases?